postgres/src/main/java/org/jdbi/v3/postgres/PostgresTypes.java

```diff
@@ -56,6 +59,20 @@ public PostgresTypes registerCustomType(Class<? extends PGobject> clazz, String
         return this;
     }
 
+    PostgresTypes setLobApi(PgLob lob) {
```

Comment:
Is the name `PgLob` appropriate? It reads like an instance of a lob, not as the handling implementation for lobs.